File: JAVA/Baju.java
//untuk, size, merk

class Baju extends Aksesoris{
    private String untuk;
    private String size;
    private String merk;

    Baju(){
        this.untuk = "";
        this.size = "";
        this.merk = "";
    };

    Baju(String id, String nama, int harga, int stok, String Jenis, String Bahan, String Warna, String Untuk, String Size, String Merk){
        setID(id);
        setNama(nama);
        setHarga(harga);
        setStok(stok);
        setJenis(Jenis);
        setBahan(Bahan);
        setWarna(Warna);
        this.untuk = Untuk;
        this.size = Size;
        this.merk = Merk;
    }

    void setUntuk(String Untuk){
        this.untuk = Untuk;
    }
    void setSize(String Size){
        this.size = Size;
    }
    void setMerk(String Merk){
        this.merk = Merk;
    }

    String getUntuk(){
        return this.untuk;
    }
    String getSize(){
        return this.size;
    }
    String getMerk(){
        return this.merk;
    }
};

File: CPP/Baju.cpp
//untuk, size, merk

#include <iostream>
#include <string>
#include "Aksesoris.cpp"

using namespace std;

class Baju : public Aksesoris{
    private:
        string untuk;
        string size;
        string merk;
    
    public:
        Baju(){
            setID("");
            setNama("");
            setHarga("");
            setStok("");
            setJenis("");
            setBahan("");
            setWarna("");
            untuk = "";
            size = "";
            merk = "";
        };

        Baju(string ID, string Nama, string Harga, string Stok, string Jenis, string Bahan, string Warna, string Untuk, string Size, string Merk){
            setID(ID);
            setNama(Nama);
            setHarga(Harga);
            setStok(Stok);
            setJenis(Jenis);
            setBahan(Bahan);
            setWarna(Warna);
            untuk = Untuk;
            size = Size;
            merk = Merk;
        };

        void setUntuk(string Untuk){
            untuk = Untuk;
        };

        void setSize(string Size){
            size = Size;
        };

        void setMerk(string Merk){
            merk = Merk;
        };

        string getUntuk(){
            return untuk;
        };

        string getSize(){
            return size;
        };

        string getMerk(){
            return merk;
        };

        ~Baju(){
        }
};